README.md

```diff
@@ -266,7 +272,21 @@ Linux amd64 Tags
 | `3.0-slim`       |  Debian 10  |
 | `3.0-appservice` |  Debian 10  |
 
-#### MCR Docs
+
+## Refreshed Dependencies
+
+Docker images are refreshed with new upstream dependencies outside of the standard release cycle to solve vulnerabilties as quickly as possible.  Refreshed images are available in two forms. 
```

Comment:
lets move this wiki  - https://github.com/Azure/azure-functions-docker/wiki/Preview:-Release-Images-for-Linux-Dedicated-(GitHub-Action)

Reply:
https://github.com/Azure/azure-functions-docker/wiki/Refreshing-Upstream-Dependencies-for-available-non-appservice-images